pipeline/translate/decoder.yml

```diff
@@ -8,6 +8,13 @@ word-penalty: 0
 mini-batch-words: 5000
 maxi-batch: 10000
 maxi-batch-sort: src
+
+# Make sure the GPU model supports half-precision decoding
+# This flag works for both Marian and CTranslate2
+precision: float16
```

Comment:
Do we have a consensus that half-precision decoding is safe to use for both student and s2s backward-models using both Marinan and CTranslate2?

Reply:
Based on the current config it was enabled for everything except CTranslate2 backward

Reply:
I would just set `fp16: true`.

Reply:
Done, but it seems it's the same thing for decoding based on Marian's docs. fp16 enables several options only in training mode.